setup.py

```diff
@@ -13,7 +13,7 @@
     "pexpect",
     "python-gnupg",
     "PyYAML",
-    "taskcluster",
+    "taskcluster<1.0.0",
```

Comment:
Yup, this was just to get the tests green on the PR.  I'm going to leave it off the final merge since we have a fixed taskcluster 1.0.2.